canonical-web-and-design/buy.ubuntu.com
# Implement design feedback

## Done

Updated the shopify theme according to design review.

## QA

- Visit https://canonical-staging.myshopify.com/
- See that it complies with the design feedback: ([part 1](https://github.com/canonical-web-and-design/buy.ubuntu.com/issues/26#issuecomment-538359241) & [part 2](https://github.com/canonical-web-and-design/web-squad/issues/1838#issuecomment-542772595))

Fixes #27
